impeller/display_list/canvas.cc

```diff
@@ -498,6 +498,11 @@ bool Canvas::AttemptDrawBlurredRRect(const Rect& rect,
     return false;
   }
 
+  // The current rrect blur math doesn't work on ovals.
+  if (fabsf(corner_radii.width - corner_radii.height) > kEhCloseEnough) {
```

Comment:
Instead of adding this case you could also adjust the logic in IsNearlySimpleRRect to force all corner radii to be equal.

Reply:
But that would also eliminate fast tessellation of simple rrects. The only deficiency here is that we can no longer fast_blur them so it's a test specific to this method.

Reply:
agh, yes don't do that then! :)